README.md

```diff
@@ -1 +1,89 @@
-# capstone_one
\ No newline at end of file
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+
+- [LIVE]
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [CapstoneProject] <a name="about-project"></a>
+[CapstoneProject] is a capstone project given by Microverse in order to help me practice what I learned in the first module. 
+
+## 🛠 Built With <a name="built-with"></a>
+Vs Code using HTML, CSS and JavaScript.
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- [Feature_1]: Setup a professional Github repository
+- [Feature_2]: Setup microverse linters
+- [Feature_3]:Use images and backgrounds to enhance the look of the website.
+- [Feature_4]: Use JavaScript to manipulate DOM elements
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: to dowload it or clone it to your machine.
+### Setup
+Clone this repository to your desired folder: https://github.com/mariegrace31/capstone_one.git
+### Install
+Install this project with: cd my-project npm install
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
You did a great job in implementing your capstone project, please add a live link showing your deployed project using github pages to enable a visitor have access to your live page from your README file 😄

Reply:
@usorfaitheloho Thanks for your review. I put the live link of my project, under the authors